### DISCUSSION TITLE
Welcome to laf Discussions!

### DISCUSSION THREAD

**Discussion by dacap** [aseprite/laf#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JustANarik** [comment]
Hello ! How do u do ? Do u remember such an issue as resizing form both center and maintain sapect ratio works weird ? Could u fix them in later update ? I hope u will :) Cheers !

**Reply by dacap** [reply]
Hi @NarayanTheProg 👋  I recommend you to create a [new discussion for new questions](https://github.com/aseprite/laf/discussions/new).

**Reply by JustANarik** [reply]
Sorry for not introducing myself and so on, i think that doesn't matter :) I use aseprite a lot, pretty much all my day, and sometimes this issue is just blowing my mind. for exmpl if I create a square, that is 32x32, then I select it and try to resize it holding both ALT and SHIFT (I'm on win 10), first thing it does, it reduces bottom and right side with one pixel, so it becomes 31x31, and that's so bad for me, and only then, it begins resizing from center, so for exmpl instead of resizing 32x32 sqr to 18x18 sqr, it resizes it to 17x17 or like that 😭 


ANOTHER ISSUE
another issue is quite  same, like when I resize a hollow rectangle or square with both ALT and SHIFT, sometimes some part of it just disappears ! 😿 

I love aseprite so much, and I don't wanna use any other software, because Aseprite is the most powerful of them. I'll be so glad if u will one day fix these issues 👍  Good bye ! Stay healthy :D

**Reply by JustANarik** [reply]
oh didn't see u'r answer. I will do that right now, thanks !

**Reply by JustANarik** [reply]
if u don't see it, 
https://github.com/aseprite/laf/discussions/25
here it is :)

Also thanks for quick answers, having fast feedback is so amazing :)
sorry for too much messages here, I'm quite new to github and I'm only 13y.o. 🐱
